tests/integration/test_kms.py

```diff
@@ -65,10 +65,6 @@ class TestKMS:
     def kms_api_snapshot_transformer(self, snapshot):
         snapshot.add_transformer(snapshot.transform.kms_api())
 
-    @pytest.fixture(scope="class")
-    def user_arn(self, aws_client):
-        return aws_client.sts.get_caller_identity()["Arn"]
-
```

Comment:
Fixture is moved out of class scope to global scope so that it can be used in all test classes.